legal-api/tests/unit/__init__.py

```diff
@@ -38,7 +38,8 @@ def has_expected_date_str_format(date_str: str, format: str) -> bool:
 
 
 @contextmanager
-def nested_session(session, exception):
+
+def nested_session(session, exception=Exception):
```

Comment:
Most calls to `nested_session` do not include the `exception` parameter, causing a error. I just set a default here to avoid this problem.